simplified-reports/src/main/java/org/nypl/simplified/reports/Reports.kt

```diff
@@ -102,7 +102,7 @@ object Reports {
       val compressedFiles =
         files.mapNotNull(this::compressFile)
       val contentUris =
-        compressedFiles.map { file -> this.mapFileToContentURI(context, file) }
+        compressedFiles.toSet().map { file -> this.mapFileToContentURI(context, file) }
 
```

Comment:
`compressedFiles` can contain duplicates. For example, if `log.txt` is compressed to `log.txt.gz`, and `log.txt.gz` already existed from a previous error report, `log.txt.gz` will appear twice in the list. The `toSet()` call removes any duplicates.